source/pkeyread.c

```diff
@@ -327,8 +327,8 @@ int main(int argc, char * const argv[])
     }
     /* run samples/formats as appropriate */
     for (k = key_id_min; k < key_id_max; k++) {
+        sample_id = k;
```

Comment:
NIT: why do we have this change here? I think it does not alter behavior of existing code at all.
I would just revert it to minimize number of files touched by this PR.

Reply:
It was just a small performance thing because sample_id only needs to be set once per loop rather than in every inner loop iteration. But I'll revert it